doc/okctl_mcp_server.md

```diff
@@ -0,0 +1,136 @@
+
+- `list_tenants()` - List all tenants
+- `create_tenant()` - Create a new tenant
+- `delete_tenant()` - Delete a specified tenant
+- `show_tenant()` - Display tenant detailed information
+- `scale_tenant()` - Scale tenant resources
+- `update_tenant()` - Update tenant configuration
+- `upgrade_tenant()` - Upgrade tenant version
+- `change_tenant_password()` - Change tenant password
+- `activate_tenant()` - Activate standby tenant
+- `replay_tenant_log()` - Replay tenant logs
+- `switchover_tenant()` - Switch between primary and standby tenants
+
+### 3. Backup Policy Management (backup_policy.py)
+
+Provides functionality for creating, deleting, viewing, updating, and managing OceanBase backup policies.
+
+- `list_backup_policies()` - List all backup policies
+- `create_backup_policy()` - Create a new backup policy
+- `delete_backup_policy()` - Delete a specified backup policy
+- `show_backup_policy()` - Display backup policy detailed information
+- `update_backup_policy()` - Update backup policy
+- `pause_backup_policy()` - Pause backup policy
+- `resume_backup_policy()` - Resume backup policy
+
+### 4. SQL Operations (sql.py)
+
+Provides functionality for configuring database connections and executing SQL queries on OceanBase clusters.
+
+- `configure_cluster_connection()` - Configure database connection to a cluster
+  - Parameters: cluster_name, tenant_name (default: "sys"), namespace (default: "default"), user, password (if not provided, will use environment variable OB_CLUSTER_PASSWORD), port (default: 2881)
+  - Returns: Database connection configuration information
+- `execute_cluster_sql()` - Execute SQL queries on a cluster
+  - Parameters: query, cluster_name (optional), tenant_name (default: "sys"), database (default: "oceanbase"), namespace (default: "default")
+  - Returns: Query results
+  - Supports various SQL commands including SELECT, SHOW TABLES, SHOW COLUMNS, DESCRIBE, and DML statements
+
+### 5. Component Management (components.py)
+
+Provides functionality for installing, updating, and managing OceanBase components.
+
+- `list_components()` - List all installed components
+- `install_component()` - Install a new component
+- `update_component()` - Update component
+
+## Development Environment Setup
+
+### Prerequisites
+
+- Python 3.10 or higher installed
+- uv package manager installed ([uv official documentation](https://github.com/astral-sh/uv))
+- OceanBase Kubernetes Control Tool (okctl) installed and configured
+- Kubernetes environment configured with access permissions to OceanBase clusters
+
+### Configuring the MCP Server
+
+```json
+{
+  "mcpServers": {
+    "okctl": {
+      "command": "uv",
+      "args": ["--directory", "path/to/mcp-oceanbase", "run", "okctl_mcp_server"],
+      "env": {
+        // you need to set these environment variables if you want to connect to cluster by root@sys
+        "OB_CLUSTER_PASSWORD": "<password of cluster>"
```

Comment:
There will be multiple obclusters running in the K8s cluster, does this config mean they should use the same password.
Besides, you can get get password from the secret.